Bootstrap/Default.ts

```diff
@@ -1,5 +1,4 @@
-import * as azureCoreAuth from "@azure/core-auth";
-
+import type { TokenCredential } from "@azure/core-auth";
```

Comment:
Please rename the PR title to something that reflect the actual change

Reply:
Updated.